machine/reference/inspect.md

```diff
@@ -61,8 +61,10 @@ For the most part, you can pick out any field from the JSON in a fairly
 straightforward manner.
 
 ```none
+{% raw %}
 $ docker-machine inspect --format='{{.Driver.IPAddress}}' dev
```

Comment:
With my PR I do not get any Liquid warnings, even after doing `jekyll clean` before `jekyll serve`. That's interesting that you see the error only in the JS console. How in the world do you know which pages to preview? How can we tell which of them have problems?